R/pkg/R/mllib.R

```diff
@@ -66,8 +67,9 @@ setClass("KMeansModel", representation(jobj = "jobj"))
 #'               \url{https://stat.ethz.ch/R-manual/R-devel/library/stats/html/family.html}.
 #' @param tol Positive convergence tolerance of iterations.
 #' @param maxIter Integer giving the maximal number of IRLS iterations.
-#' @return a fitted generalized linear model
+#' @return \code{spark.glm} returns a fitted generalized linear model
```

Comment:
I'd view `spark.glm` as a method under this group. It might be clearer to have the method name in the `Value` section. Also, I'm not sure about how roxygen2 defines the ordering. Having the names would help.